guides/tutorial/start-tutorial.md

```diff
@@ -102,9 +102,9 @@ defmodule AuthMe.UserManager.Guardian do
   end
 
   def resource_from_claims(%{"sub" => id}) do
-    case UserManager.get_user(id) do
-      nil -> {:error, :resource_not_found}
-      user -> {:ok, user}
+    case UserManager.get_user!(id) do
```

Comment:
I believe we'll need to update this function a bit more as a result of shift to the bang function. 

In Elixir the convention is for non-bang (ex. `get_user/1`) to return `{:ok, result}` or `{:error, reason}`, for which this `case` would work perfectly. However, since Phoenix appears to no longer generate a non-bang we need to take into account the difference: these bang functions (`get_user!/1`) return `result` or raise an error; under-the-hood the Phoenix code uses [Repo.one!/1](https://hexdocs.pm/ecto/Ecto.Repo.html#c:one!/2).

To accommodate this difference I believe we'll want something like:

```elixir
def resource_from_claims(%{"sub" => id}) do
  user = UserManager.get_user!(id)
  {:ok, user}
rescue 
  e in Ecto.NoResultsError => {:error, :resource_not_found}
end
```

What are your thoughts @yordis, @Hanspagh?

Reply:
I agree with @doomspork, sorry for the misleading advise in the old PR

Reply:
I agree with @doomspork, sorry for the misleading advise in the old PR